code/core/src/phone/java/com/adobe/marketing/mobile/services/SQLiteDataQueue.java

```diff
@@ -121,26 +122,22 @@ public List<DataEntity> peek(final int n) {
                                     ServiceConstants.LOG_TAG,
                                     LOG_PREFIX,
                                     String.format(
-                                            "query - Successfully read %d rows from table(%s)",
-                                            rows.size(), TABLE_NAME));
+                                            "query - Successfully read %d rows from table.",
+                                            rows.size()));
                             return true;
                         } catch (final SQLiteException e) {
                             Log.warning(
                                     ServiceConstants.LOG_TAG,
                                     LOG_PREFIX,
                                     String.format(
-                                            "query - Error in querying database table (%s). Error:"
+                                            "query - Error in querying database table. Error:"
                                                     + " (%s)",
-                                            TABLE_NAME, e.getLocalizedMessage()));
+                                            e.getLocalizedMessage()));
                             return false;
                         }
                     });
         }
 
-        if (rows.isEmpty()) {
-            return new ArrayList<>();
-        }
-
```

Comment:
Removed this clause so that the trace log below is printed for the case where no data entities are returned.